presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
   This is the HTTP request log which contains every HTTP request
   received by the server. It is automatically rotated and compressed.
 
+An Example Deployment on Laptop
+-------------------------------
+
+This section shows how to run Presto connecting to Hive MetaStore on a single laptop to query data in an S3 bucket.
+
+Configure Hive MetaStore
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Download and extract the binary tarball of Hive.
+For example, download `apache-hive-2.3.7-bin.tar.gz <https://downloads.apache.org/hive/hive-2.3.7/apache-hive-2.3.7-bin.tar.gz>`_ .
+
+.. code-block:: console
+
+    $ tar -zxf apache-hive-2.3.7-bin.tar.gz
+    $ cd apache-hive-2.3.7-bin
+
+You only need to launch Hive Metastore to serve Presto catalog information such as table schema and partition location.
+If it is the first time to launch the Hive Metastore, prepare corresponding configuration files and environment, also initialize a new Metastore:
+
+.. code-block:: console
+
+    $ export HIVE_HOME=`pwd`
+    $ cp conf/hive-default.xml.template conf/hive-site.xml
+    $ mkdir -p hcatalog/var/log/
+    # only required for the first time
+    $ bin/schematool -dbType derby -initSchema
+
+If you want to access AWS S3, append the following lines in ``conf/hive-env.sh``.
+Hive needs the corresponding jars to access files with ``s3a://`` addresses, and AWS credentials as well to access an S3 bucket (even it is public).
+If you do not have a Hadoop installation which include above jars, download them from `maven central repository <https://repo1.maven.org/>`_.
+
+.. code-block:: console
+
+    $ export HIVE_AUX_JARS_PATH=${HADOOP_HOME}/share/hadoop/tools/lib/aws-java-sdk-core-1.10.6.jar:${HADOOP_HOME}/share/hadoop/tools/lib/aws-java-sdk-s3-1.10.6.jar:${HADOOP_HOME}/share/hadoop/tools/lib/hadoop-aws-2.8.4.jar
```

Comment:
omit jar versions?
are these paths static and reliable?

Reply:
updated